lib/modules/manager/sbt/__fixtures__/scala-version-variable.sbt

```diff
@@ -1,7 +1,8 @@
 val ScalaVersion = "2.12.10"
 val versionExample = "0.0.8"
+val fileVersion = "3.2.1"
 
-version := "3.2.1"
+version := fileVersion
```

Comment:
why this change?

Reply:
`scalaVersion` can be set via variable defined earlier, so we should support this also for `version` (which translates to `packageFileVersion` in Renovate).

Reply:
ok, so this case is still supported? just wanted to make sure 🤗

Reply:
Sure, I think I'll return this one back and create simpler separate test for the new case